website/docusaurus.config.js

```diff
@@ -81,6 +81,17 @@ module.exports = {
         version: "0.1.0",
         releaseDate: "2021-06-23",
         firstDocPath: "/development",
+      },
+      {
+        name: "APISIX™ Go Plugin Runner",
+        nameInParamCase: "go-plugin-runner",
+        description: "Runs Apache APISIX plugins written in Go. Implemented as a sidecar that accompanies Apache APISIX.",
+        shape: "star",
```

Comment:
changing the shape will be better, not urgent I can do that on next PR

Reply:
@iamayushdas 
What shape do you recommend?

Reply:
nonagon

Reply:
Updated.

Reply:
no its not, sir you have to add a shape(nonagon).png to reflect it on doc page

Reply:
<img width="1135" alt="image" src="https://user-images.githubusercontent.com/40708551/124604831-f73fbc80-de88-11eb-8c5a-77af6db84fb7.png">

Reply:
It's difficult for contributors who are not familiar with those configurations, I think it's ok to use Star here, and @iamayushdas you could help to submit a new PR to update the shape.

Reply:
okay sir

Reply:
#404